docs/physics/transport/braginskii.rst

```diff
@@ -21,3 +21,4 @@ Reference/API
 .. automodapi:: plasmapy.physics.transport.braginskii
    :no-heading:
    :no-main-docstr:
+   :skip: Coulomb_logarithm grab_charge
```

Comment:
Should we make `grab_charge` a private function by renaming it as `_grab_charge` if it doesn't need to be part of the public API?

Reply:
Test!

Reply:
Darn autocorrect to heck! I meant to say yes. :D